### PR TITLE
Feat/cli test

### PR DESCRIPTION
**Pull request by Sturla**
# What?

A high-level description.  What are you changing?
Tests for Cli.java
# Why?

Why are you doing this?  What is the business value of this change?
For increased Code Coverage
# Reviewer checklist

[ ] - The change introduces no untested code
[ ] - The change passes all existing tests
[ ] - The change introduces no typos or formatting errors

### CODE REVIEW

File: src/main/java/is/ru/tictactoe/Cli.java [hunk 1]
```diff
@@ -69,11 +71,23 @@ private void greeting() {
 	// Translate index to x,y coordinates
 	private coordinates getCellCords(int index) {
 		coordinates cord = new coordinates();
-		cord.x = (index % 3);
-		cord.y = (index / 3);
+	
+		cord.x = setCordx(index);	
+		cord.y = setCordy(index);	
+	//	cord.x = (index % 3);
```
Comment (eirikura) [inline src/main/java/is/ru/tictactoe/Cli.java]: This can be deleted, right?

File: src/main/java/is/ru/tictactoe/Cli.java [hunk 2]
```diff
@@ -107,7 +122,7 @@ private void makeMove(int moves) {
 
 	private boolean playAgain() {
 		println("Play again? (y/n): ");
-		while (!sc.hasNext("[yn]")) {
+		    while (!sc.hasNext("[yn]")) {
```
Comment (eirikura) [inline src/main/java/is/ru/tictactoe/Cli.java]: This indent is weird?

File: src/main/java/is/ru/tictactoe/Cli.java [hunk 3]
```diff
@@ -23,15 +23,17 @@ public Cli() {
...
+		System.out.print(string);
 	}
 	
 	private void println(String string) {
-		System.out.println(string);;
+		System.out.println(string);
 	}
 
-	private void printBoard() {
+	// changed to public
```
Comment (solvip) [inline src/main/java/is/ru/tictactoe/Cli.java]: No need for this comment

File: src/main/java/is/ru/tictactoe/Cli.java [hunk 4]
```diff
@@ -69,11 +71,23 @@ private void greeting() {
...
+	//	cord.x = (index % 3);
+	//	cord.y = (index / 3);
 		return cord;
 	}
 
+	public int setCordx(int index) {
+		return (index % 3);
+	}
+
+	public int setCordy(int index) {
```
Comment (solvip) [inline src/main/java/is/ru/tictactoe/Cli.java]: I feel that the names of these two functions are wrong.  The 'set' part indicates that they're setting something, but they're completely side effect free.

Maybe they should simply be called `cordX` and `cordY`?

File: src/main/java/is/ru/tictactoe/Cli.java (hunk 2 above)
Comment (solvip) [inline src/main/java/is/ru/tictactoe/Cli.java]: nit: the while shouldn't be indented

File: src/test/java/is/ru/tictactoe/CliTest.java [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+import org.junit.Assert;
+import org.junit.Test;
+
+public class CliTest {
+	
+	@Test
+	public final void testPlayerGetsNewName() {
+		String name = "NN";
+		Player player = new Player(name);
+		assertEquals("NN", player.name);
```
Comment (solvip) [inline src/test/java/is/ru/tictactoe/CliTest.java]: Nit: I feel that this should either read:

```
String name = "NN";
Player player = new Player(name);
assertEquals(name, player.name);
```

or 

```
Player player = new Player("NN");
assertEquals("NN", player.name);
```

**Review by solvip — COMMENTED** [review]
Couple of remarks, otherwise fine!

File: src/main/java/is/ru/tictactoe/Cli.java (hunk 2 above)
Comment (Sturla) [reply src/main/java/is/ru/tictactoe/Cli.java]: not intentional

File: src/main/java/is/ru/tictactoe/Cli.java (hunk 3 above)
Comment (Sturla) [reply src/main/java/is/ru/tictactoe/Cli.java]: note to self I forgot to delete

File: src/main/java/is/ru/tictactoe/Cli.java (hunk 1 above)
Comment (Sturla) [reply src/main/java/is/ru/tictactoe/Cli.java]: yeah of course thought I had cleaned everything up

File: src/main/java/is/ru/tictactoe/Cli.java (hunk 4 above)
Comment (Sturla) [reply src/main/java/is/ru/tictactoe/Cli.java]: I agree

**Review by solvip — APPROVED** [review]
Goodstuff.  Ship it.

**Pull request closed without merging**